### PR TITLE
29 cockpit docs plugin

### PR DESCRIPTION
**Pull request by jbbjarnason**

### CODE REVIEW

File: vcpkg-overlays/triplets/x64-linux-gcc.cmake [hunk 1]
```diff
@@ -3,6 +3,10 @@ set(VCPKG_LIBRARY_LINKAGE static)
 set(VCPKG_CRT_LINKAGE dynamic)
 set(VCPKG_CMAKE_SYSTEM_NAME Linux)
 
+#set(CMAKE_EXE_LINKER_FLAGS "-static-libgcc -static-libstdc++ -static")
```
Comment (jbbjarnason) [inline vcpkg-overlays/triplets/x64-linux-gcc.cmake:6]: remove changes

File: .github/workflows/static-packing.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          # id must be checkout from a Git repo. It must not set if using a submodule
+          # for vcpkg.
+          vcpkgGitCommitId: '19af97cba8ca48474e4ad15a24ed50271a9ecdac'
+          vcpkgJsonGlob: 'vcpkg.json'
+
+      # todo would be nice to use lukka/run-cmake but rpm needs root for some reason when packaging
+      - name: Pack it
+        run: |
+          cmake --workflow --preset packflow-gcc-${{matrix.build_type}}-static
+        shell: alpine.sh --root {0}
```
Comment (omarhogni) [inline .github/workflows/static-packing.yml:78]: Look into building rpms without needing root.

File: .github/workflows/static-packing.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+      - name: Test install
+        run: |
+          ls build
```
Comment (omarhogni) [inline .github/workflows/static-packing.yml:82]: Debug primative left.

File: .github/workflows/static-packing.yml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build_static_artifacts:
+    runs-on: ubuntu-22.04
```
Comment (omarhogni) [inline .github/workflows/static-packing.yml:11]: Run this as a matrix of 18.04 and 20.04 also

File: CMakePresets.json [hunk 5]
```diff
@@ -128,23 +185,33 @@
...
+    {
+      "name": "pack-gcc-release-static",
+      "configurePreset": "pack-gcc-release-static",
       "inherits": [ "pack-linux-common" ]
     }
   ],
   "workflowPresets": [
     {
-      "name": "buildflow-docs",
+      "name": "workflow-docs",
```
Comment (omarhogni) [inline CMakePresets.json:210]: Possibly breaks docs pipeline.

File: docs/manifest.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
+  "version": 0,
+  "tools": {
+    "pinger": {
```
Comment (omarhogni) [inline docs/manifest.json:4]: change pinger name for cockpit manifest.

File: libs/logger/src/logger.cpp [hunk 7]
```diff
@@ -25,7 +25,7 @@ PRAGMA_CLANG_WARNING_POP
 
 tfc::logger::logger::logger(std::string_view key) : key_{ key } {
   // Create sinks
-  std::vector<spdlog::sink_ptr> sinks{ std::make_shared<spdlog::sinks::tfc_systemd_sink_mt>(key_) };
+  std::vector<spdlog::sink_ptr> sinks{ std::make_shared<spdlog::sinks::stderr_color_sink_mt>() };
```
Comment (omarhogni) [inline libs/logger/src/logger.cpp:28]: This propably double logs to the console when --stdout is enabled.

**Review by omarhogni — APPROVED** [review]

File: .github/workflows/static-packing.yml (hunk 2 above)
Comment (jbbjarnason) [reply .github/workflows/static-packing.yml:75]: https://github.com/Skaginn3x/framework/issues/36

File: .github/workflows/static-packing.yml (hunk 3 above)
Comment (jbbjarnason) [reply .github/workflows/static-packing.yml]: removed

File: CMakePresets.json (hunk 5 above)
Comment (jbbjarnason) [reply CMakePresets.json:210]: fixed

File: libs/logger/src/logger.cpp (hunk 7 above)
Comment (jbbjarnason) [reply libs/logger/src/logger.cpp]: removed double logs, this will need to be reverted

**Pull request merged**